web/src/routes/__tests__/PoisPage.spec.tsx

```diff
@@ -1,31 +1,23 @@
 import { fireEvent } from '@testing-library/react'
+import { mocked } from 'jest-mock'
 import React from 'react'
-import { ThemeProvider } from 'styled-components'
 
-import { cityContentPath, CityModelBuilder, LanguageModelBuilder, PoiModelBuilder, POIS_ROUTE } from 'api-client'
 import {
-  mockUseLoadFromEndpointWithData,
-  mockUseLoadFromEndpointWithError
-} from 'api-client/src/testing/mockUseLoadFromEndpoint'
+  cityContentPath,
+  CityModelBuilder,
+  LanguageModelBuilder,
+  PoiModelBuilder,
+  POIS_ROUTE,
+  prepareFeatureLocations
+} from 'api-client'
 
-import { mockGeolocationSuccess } from '../../__mocks__/geoLocation'
-import buildConfig from '../../constants/buildConfig'
+import { useFeatureLocations } from '../../hooks/useFeatureLocations'
 import { renderWithRouter } from '../../testing/render'
 import PoisPage from '../PoisPage'
 
-jest.mock('api-client', () => ({
-  ...jest.requireActual('api-client'),
-  useLoadFromEndpoint: jest.fn()
-}))
-
-jest.mock('react-i18next', () => ({
-  useTranslation: () => ({
-    t: (key: string) => key
-  })
-}))
-
-// @ts-expect-error -- ignore readOnly var
-navigator.geolocation = mockGeolocationSuccess
+jest.mock('react-i18next')
+jest.mock('../../utils/getUserLocation', () => async () => ({ status: 'ready', coordinates: [10.8, 48.3] }))
```

Comment:
seems like this mock does not work, as the distance wasn't rendered if you `debug()`